packages/universal-components/src/RangeDatePicker/components/RenderDay.js

```diff
@@ -147,10 +136,12 @@ export default class RenderDay extends React.Component<Props, State> {
       >
         <View>
           {isRangePicker &&
+            isPossibleToChangeDateThisDirection &&
             (isStartOfSelectedDates || this.state.isDragging) && (
```

Comment:
This big condition statement scares me :), there must be a way how to write this in a more readable way.

Reply:
I tried to make it better. But it's still quite complicated, it has to solve lot of cases.